### DISCUSSION TITLE
SOLWEIG-processing for a mosaic of Rasters

### DISCUSSION THREAD

**Discussion by jonas260492** [UMEP-dev/UMEP#276] [Q&A] [answered]
Hey
thanks one more time for the tool.

I have some unexpected results when calculating Mean Radiant Temperature (Tmrt) in a loop with a script.
I divided the city in a mosaic of ca. 9.500 elements (100x100meter) because I count with fine level meteorological data from sensors and I want to ensure that weather data is as fine as possible. So I have different weatherdata for all of these mosaic elements. For the mosaic elements I have all necessary data for SOLWEIG-tool on 2.5x2.5meter resolution. 
When I compose the 9.500 elements after processing to one image which I need for my further analysis, I see that at the borders of the elements, Tmrt is very high which does not make sense because also in areas with vegetations this is the case (see image). 

All of the 9.500 input elements are continuous if I put them next to each other before processing. Just the Tmrt output has this wrong result.

Is there a way to avoid this wrong outcome? 

![image](https://user-images.githubusercontent.com/59968564/114866801-4fb86e00-9df4-11eb-9f26-a9b7f3d1ec90.png)
Windows 10 Pro. I installed QGIS as described in the UMEP manual. UMEP for processing version 1.3. 

Maybe the settings for the SOLWEIG are from interest:
processing.run("umep:Outdoor Thermal Comfort: SOLWEIG",{
    'ABS_L' : 0.97,
    'ABS_S' : 0.7,
    'ACTIVITY' : 80, 
    'AGE' : 35,
    'ALBEDO_GROUND' : 0.15,
    'ALBEDO_WALLS' : 0.2,
    'CLO' : 0.9,
    'CYL' : True,
    'EMIS_GROUND' : 0.95,
    'EMIS_WALLS' : 0.9,
    'HEIGHT' : 180, 
    'INPUTMET' : inpath_meteo,
    'INPUT_ANISO' : '',
    'INPUT_ASPECT' : inpath_WAspect,
    'INPUT_CDSM' : inpath_nDOMv,
    'INPUT_DEM' :inpath_DEM,
    'INPUT_DSM' : inpath_DOM,
    'INPUT_HEIGHT' : inpath_WHeight,
    'INPUT_LC' : inpath_LaCov,
    'INPUT_SVF' : inpath_SVF,
    'INPUT_TDSM' : None,
    'INPUT_THEIGHT' : 25, 
    'ONLYGLOBAL' : True, 
    'OUTPUT_DIR' : outpath,
    'OUTPUT_KDOWN' : False,
    'OUTPUT_KUP' : False,
    'OUTPUT_LDOWN' : False, 
    'OUTPUT_LUP' : False, 
    'OUTPUT_SH' : False,
    'OUTPUT_TMRT' : True,
    'OUTPUT_TREEPLANTER' : False, 
    'POI_FIELD' : '',
    'POI_FILE' : None,
    'POSTURE' : 0, 
    'SAVE_BUILD' : False, 
    'SENSOR_HEIGHT' : 10, 
    'SEX' : 0, 
    'TRANS_VEG' : 3, 
    'USE_LC_BUILD' : False, 
    'UTC' : 1, 
    'WEIGHT' : 75 })

Thank you very much,
jonas

**Comment by biglimp** [comment, marked as answer]
Thanks @jonas260492. The reason that you see edge effects is that shadows from adjacent areas, both when sky view factor is calculated and when SOLWEIG is executed related to shadowing from the Sun. The solution would be to tile with and overlap e.g. 50 meter on each sides.

**Reply by jonas260492** [reply]
Thank you @biglimp , I see. That makes sense!
